src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -157,6 +157,11 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void updateFilteredLocationList(Predicate<Location> predicate) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```

Comment:
Considering the nature of this test, it might be best that the method updateFilteredLocationList is completely removed from the working program since it is not expected to be called.

Reply:
Additionally, on this aspect, I realised that updateFilteredLocationList is a method newly added to Model and ModelManager. Considering that, may I check why is the method added if it is not intended to be called as seen from this test?

Reply:
The method was added for the following reasons:
1. The Person and Visit classes both have these two methods in Model - getFiltered[x]List() and updateFiltered[x]List() where x is either of those classes. As the InfoHandler class requires the getFilteredLocationList(), I felt it was apt that updateFilteredLocationList() was added as well.
2. Also, the addLocation method in the ModelManager requires the updateFilteredLocationList() as well.

As for the test above, the function is required to be overwritten for the ModelStub to be created.

Reply:
I understand now. Thank you for the clarification.